src/main/java/uk/gov/ons/census/action/builders/PrintFileDtoBuilder.java

```diff
@@ -17,7 +17,8 @@
           ActionType.P_RL_2RL1A,
           ActionType.P_RL_2RL2BA,
           ActionType.P_RL_1IRL1,
-          ActionType.P_RL_1IRL2B
+          ActionType.P_RL_1IRL2B,
```

Comment:
`P_RL_1IRL2B` - this one also shouldn't be on the list. It must have been added by somebody else by accident

Reply:
`P_RL_1IRL2B` Should be here, it's the individual `IRL` code.

Reply:
> `P_RL_1IRL2B` - this one also shouldn't be on the list. It must have been added by somebody else by accident

Nick as requested I have removed P_RL_1IRL2B from the list.

Reply:
Nick - spoke to Adam on this one are you comfortable with P_RL_1IRL2B being in the list?

Reply:
Fine, leave it